locksmith/package.json

```diff
@@ -98,6 +100,7 @@
     "@types/cors": "2.8.17",
     "@types/geoip-country": "4.0.2",
     "@types/isomorphic-fetch": "0.0.39",
+    "@types/jsonwebtoken": "^9",
```

Comment:
Please use a specific version here to ensure consistent builds

Reply:
an oversight! fixed!